Exercises/exercise_2_btc_transfer_with_c2d2.md

```diff
@@ -98,12 +97,12 @@ The contract will show in metamask as symbol 'Satoshi'. If your recipient addres
 1. Generate an ethereum withdrawal address. The Bitcoin address you provide will be uniquely linked to the deposit address and receive the withdrawn BTC on the Bitcoin testnet. 
 
    ```
-   `c2d2cli` withdraw-btc ethereum [bitcoin recipient address] [fee]
+   c2d2cli withdraw-btc ethereum [bitcoin recipient address] [fee]
    ```
 
 For example:
    ```
-   `c2d2cli` withdraw-btc ethereum tb1qg2z5jatp22zg7wyhpthhgwvn0un05mdwmqgjln 10000satoshi
+   c2d2cli withdraw-btc ethereum tb1qg2z5jatp22zg7wyhpthhgwvn0un05mdwmqgjln 10000satoshi
```

Comment:
the fee amount that I put originally might be too large, lower it to 1000satoshi should be good enough?